src/tree/kat_encryption.rs

```diff
@@ -329,12 +328,10 @@ fn generate_test_vectors() {
     }
 
     for ciphersuite in Config::supported_ciphersuites() {
-        for n_leaves in 1u32..50 {
+        for n_leaves in 1u32..9 {
```

Comment:
I'd prefer keeping more in here. But I see that it creates issues when re-generating. I'll make these things configurable.